src/test/java/io/crate/consumer/CrateConsumer.java

```diff
@@ -77,7 +73,15 @@ public void accept(Result result) throws IOException {
                     System.currentTimeMillis(),
                     benchmarkValues(result)
             };
-            client.sql(new SQLRequest(CrateConsumerConstants.INSERT_STMT, args)).actionGet();
+            JsonObject content = new JsonObject();
+            content.add("args", gson.toJsonTree(args));
+            content.addProperty("stmt", CrateConsumerConstants.INSERT_STMT);
+            HttpPost request = new HttpPost(composeURI(getHost(), getPort(CrateConsumerConstants.CRATE_HTTP_PORT)));
+            request.setEntity(new StringEntity(gson.toJson(content)));
+            HttpResponse res = httpClient.execute(request);
```

Comment:
we can extract lines from 76 to 80 to the separate method, e.g. ``prepareCrateHttpRequest``, coz lots of stuff is going on in this method already.